src/main/scala/tech/cryptonomic/conseil/tezos/ApiOperations.scala

```diff
@@ -185,6 +186,51 @@ object ApiOperations {
                   (String, String, String, Boolean, Boolean, Option[String], Int, Option[String], BigDecimal),
                   Seq]) extends Action
 
+  /**
+    * Trait which enumerates the types of operations that can be performed between parties.
+    */
+  sealed trait OperationKind
+
+  case class SeedNonceRevelation() extends OperationKind
```

Comment:
Can we replace multiple case classes with an enum?

Reply:
Oh, this ended up never being used, because I didn't refactor how we filter at all